config/software/ruby.rb

```diff
@@ -233,6 +233,7 @@
     # force that API off.
     configure_command << "ac_cv_func_arc4random_buf=no"
   elsif windows?
+    env["CC"] = "gcc"
```

Comment:
Can you slap a comment on why we added this. It's helpful years later when we're cleaning these up